### DISCUSSION TITLE
upgrade problems

### DISCUSSION THREAD

**Discussion by ander841** [SpiralLinux/SpiralLinux-project#416] [General]
upgrade problems. it seems my old kernel will not leave for the new one. doing my upgrades with synaptic it wont let me install any new upgrades.

E: linux-image-6.1.0-12-amd64: installed linux-image-6.1.0-12-amd64 package post-removal script subprocess returned error exit status 1                                                                      dpkg: error processing package linux-image-6.5.0-0.deb12.4-amd64 (--configure):
 installed linux-image-6.5.0-0.deb12.4-amd64 package post-installation script subprocess returned error exit status 1
dpkg: dependency problems prevent configuration of linux-image-amd64:
 linux-image-amd64 depends on linux-image-6.5.0-0.deb12.4-amd64 (= 6.5.10-1~bpo12+1); however:
  Package linux-image-6.5.0-0.deb12.4-amd64 is not configured yet.

dpkg: error processing package linux-image-amd64 (--configure):
 dependency problems - leaving unconfigured
Processing triggers for initramfs-tools (0.142) ...
update-initramfs: Generating /boot/initrd.img-6.5.0-0.deb12.4-amd64
/usr/bin/ldd: line 119: printf: write error: Broken pipe
W: Possible missing firmware /lib/firmware/i915/dg2_huc_gsc.bin for module i915
W: Possible missing firmware /lib/firmware/i915/mtl_huc_gsc.bin for module i915
W: Possible missing firmware /lib/firmware/i915/mtl_guc_70.bin for module i915
raspi-firmware: missing /boot/firmware, did you forget to mount it?
run-parts: /etc/initramfs/post-update.d//z50-raspi-firmware exited with return code 1
dpkg: error processing package initramfs-tools (--configure):
 installed initramfs-tools package post-installation script subprocess returned error exit status 1
Errors were encountered while processing:
 linux-image-6.5.0-0.deb12.4-amd64
 linux-image-amd64
 initramfs-tools

E: linux-image-6.1.0-12-amd64: installed linux-image-6.1.0-12-amd64 package post-removal script subprocess returned error exit status 1

**Comment by geckolinux** [comment]
https://github.com/SpiralLinux/SpiralLinux-project/issues/413
